bears/general/InvalidLinkBear.py

```diff
@@ -33,14 +33,20 @@ def find_links_in_file(file):
             match = regex.search(line)
             if match:
                 link = match.group()
-                code = InvalidLinkBear.get_status_code_or_error(link)
+                code = InvalidLinkBear.get_status_code_or_error(link, timeout)
                 yield line_number + 1, link, code
 
-    def run(self, filename, file):
+    def run(self, filename, file, timeout: str):
```

Comment:
I didn't set it to`int` because I had to check if the value was supplied by user or not. `string` gets a value of `""` if no value is provided by user. 

> And give a default value here ?

If you a set a default there, then there is no prompt for a value. The value of `timeout` is always 2